lib/kubecfg.libsonnet

```diff
@@ -56,4 +56,10 @@
   // to refer to submatches.  Regex is as implemented in golang regexp
   // package (python-ish).
   regexSubst:: std.native("regexSubst"),
+
+  // parseHelmChart(chartData, releaseName, namespace, values): Expand
```

Comment:
what about renaming it to `manifestHelm` ?

Reply:
brain fart. yeah `manifestXX` is used to convert jsonnet into XX.

I understand that it makes sense to use `parse` to convey that it takes an external format and it "raises" it to jsonnet.

The thing that made confused me though is that we're not literally parsing a helm chart and presenting it as jsonnet; instead we're running a helm chart template expansion and parsing its result.

`parseRenderedHelmChyart` would be keyboardful

what about `renderHelmChart` or something similar?